000-vip-init.php

```diff
@@ -9,6 +9,24 @@
  * good reason not to.
  */
 
+if ( defined( 'VIP_GO_SITE_BLOCKED' ) && VIP_GO_SITE_BLOCKED ) {
+	if ( '/cache-healthcheck?' === $_SERVER['REQUEST_URI'] ) {
```

Comment:
Can we avoid the duplication here somehow?

Reply:
The tricky thing there is the 'regular' code comes from `WPCOM_VIP_Cache_Manager::__construct()`, which gets called when `vip-cache-manager.php` is included.

We could add our code after that mu-plugin gets loaded, the only downside is it's fairly 'late' (lots of other setup happens in between) and we'd essentially need a way to ensure our new code was loaded after it, something like `zzz-vip-init.php`.

Reply:
Maybe we can just move the healthcheck out of the cache manager plugin, and into the top of `001-vip-init.php`?

Reply:
Moving the whole check probably makes sense. There was some initial work on that here: https://github.com/Automattic/vip-go-mu-plugins/pull/386 (although, that changes some of the logic as well which we can ignore for the purposes of this PR).